src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -0,0 +1,97 @@
+
+<img
+  title="Comment on any page "
+  alt="A screenshot of the UI that depicts commenting on a specific page."
+  src={green_dot}
+/>
+
+If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, click the icon to open the comment panel. Type your comment and start a discussion on the page you're looking at. When you comment on a dashboard or an error in New Relic, that comment stays with your data. You don't need to spend time repeating information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
+
+You may notice the “add comment” icon is disabled on some pages. By design, in a multi-account organization you can't discuss pages that either don't have data associated with a specific account. You also won't be able to discuss on pages that present data belonging to multiple accounts in a single view. Simply selecting a specific account or clicking through to a specific item or entity will usually show the “add comment” icon.
+
+## At-mention your teammates and notify by email [#at-mention-your-teammates]
+
+Like other collaboration tools, when you at-mention (@) your teammates in a comment, they'll be notified by email. If you at-mention a teammate that isn't part of the preview or isn't using the new redesign, they won't have access to the discussion on New Relic. 
+
+## Share links and screenshots with Slack [#sync-with-slack]
+
+Whenever you start a new discussion, you can sync it with Slack so you can collaborate with all your teammates, even those that aren't on New Relic.
+
+Once the Slack app is configured and you are authenticated, select the Slack channel where you want your discussion synced. If you regularly use the same Slack channel it'll be prefilled so no extra clicks needed. From Slack, any teammate, even if they are not a New Relic user, can reply and their comment will be brought back into the thread on New Relic. Likewise subsequent replies added to the thread on New Relic will also be added to the thread on Slack.
+
+[Learn more about the Slack integration](https://docs-preview.newrelic.com/docs/collaboration-slack-integration).
+
+## Built-in screenshot sharing [#built-in-screenshots]
+
+With a new integrated screenshot selection tool you can attach screenshots to your comments. Click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
+
+## Close and re-open discussions [#close-reopen]
+
+Once a question is answered or issue is resolved you can mark a discussion as “Closed” via the more actions menu on the discussion thread. Similar to a Jira ticket your team can still add comments to the discussion. This will not automatically re-open a discussion but if you need to you can manually do so the same way as closing.
```

Comment:
I think there should be an "an" before "issue" in "issue is resolved", which should also have a comma after the "resolved". I'm also not 100% sure on the styling yet but should there be something indicating "more actions" as a thing the user interacts with?

Reply:
I think the second sentence should be flipped around: Your team can still add comments to the discussion similar to a Jira ticket.

Reply:
I think there should be a comma after "discussion" in the third sentence, and you can also probably cut out "if you need to you". Also the "closing" at the end should be followed by what they are closing.